### PR TITLE
Change Font color for visibility (#4094)

### PR DESCRIPTION
**Pull request by chrlschwb**
fixes issue #4094

### CODE REVIEW

File: packages/markdown-editor/src/MarkdownEditor.js [hunk 1]
```diff
@@ -90,3 +90,4 @@ export default {
   ClassicEditor,
   InlineEditor,
 }
+
```
Comment (traumschule) [inline packages/markdown-editor/src/MarkdownEditor.js:93]: This change seems unnecessary.

**Review by traumschule — CHANGES_REQUESTED** [review]
LGTM, except

**Review by oleksanderkorn — APPROVED** [review]

**Review by thesan — COMMENTED** [review]
@chrlschwb LGTM!

Just please ~~accept @oleksanderkorn suggestion~~ see comment :point_down:

**Review by chrlschwb — COMMENTED** [review]
delete empty line

**Review by thesan — APPROVED** [review]

**Pull request merged**